pkg/sql/plan/apply_indices.go

```diff
@@ -319,7 +319,7 @@ END0:
 			}
 			idxFilter, _ = BindFuncExprImplByPlanExpr(builder.GetContext(), funcName, []*plan.Expr{
 				{
-					Typ: DeepCopyType(idxTableDef.Cols[0].Typ),
+					Typ: *DeepCopyType(idxTableDef.Cols[0].Typ),
```

Comment:
This one, Typ: *DeepCopyType(XXX),          isn't this just Typ: XXX?

Reply:
done